programs/drift/src/math/spot_withdraw.rs

```diff
@@ -15,6 +15,8 @@ pub fn calculate_min_deposit_token_amount(
     deposit_token_twap: u128,
     withdraw_guard_threshold: u128,
 ) -> DriftResult<u128> {
+    // minimum deposits permitted for an immediate withdraw
+    // minimum deposit amount lower of 75% of TWAP or withdrawal guard threshold below TWAP
```

Comment:
as far as i understand it, the state space contains 3 scenarios:
1) the twap is above the withdraw guard threshold and the twap / 4 > withdraw guard threshold, therefore the min deposit token amount is .75 * deposit_token_twap
2) the twap is above the withdraw guard threshold and the twap / 4 < withdraw guard threshold, therefore the min deposit token amount is deposit_token_twap - withdraw_guard_threshold
3)  the twap is below the withdraw guard threshold, therefore the min deposit is 0

feel like having the comment outline these three scenarios is better, cuz your comment would lead the (inattentive) reader to assume there are only two scenarios

Reply:
i attempted to satisfy this